Description: use Test::CPAN::Meta::YAML::Version instead of the vanished Test::YAML::Meta::Version
Origin: https://rt.cpan.org/Public/Bug/Display.html?id=65903
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=65903
Bug-Debian: http://bugs.debian.org/621468
Forwarded: https://rt.cpan.org/Public/Bug/Display.html?id=65903
Author: Jerome Quelin <jquelin@cpan.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-04-07

--- a/Build.PL
+++ b/Build.PL
@@ -15,7 +15,7 @@
         'Module::Pluggable'         => '2.96',
         'IO::Capture'               => '0.05',
         'Array::Diff'               => '0.04',
-        'Test::YAML::Meta::Version' => '0.11',
+        'Test::CPAN::Meta::YAML::Version' => '0.11',
         'version'                   => '0.73',
         'List::MoreUtils'           => '0',
         'File::Find::Rule'          => '0',
--- a/lib/Module/CPANTS/Kwalitee/MetaYML.pm
+++ b/lib/Module/CPANTS/Kwalitee/MetaYML.pm
@@ -3,7 +3,7 @@
 use strict;
 use File::Spec::Functions qw(catfile);
 use YAML::Syck qw(Load LoadFile);
-use Test::YAML::Meta::Version;
+use Test::CPAN::Meta::YAML::Version;
 
 sub order { 20 }
 
@@ -149,7 +149,7 @@
         }
     }
     $hash{spec} = $version;
-    my $spec = Test::YAML::Meta::Version->new(%hash);
+    my $spec = Test::CPAN::Meta::YAML::Version->new(%hash);
     if ($spec->parse()) {
         my $report_version= $version || 'known';
         my @errors;
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -25,7 +25,7 @@
                            'Test::More' => '0',
                            'Test::NoWarnings' => '0',
                            'Test::Warn' => '0.11',
-                           'Test::YAML::Meta::Version' => '0.11',
+                           'Test::CPAN::Meta::YAML::Version' => '0.11',
                            'Text::CSV_XS' => '0.45',
                            'YAML::Syck' => '0.95',
                            'version' => '0.73'
--- a/META.yml
+++ b/META.yml
@@ -27,7 +27,7 @@
   Pod::Simple::Checker: 2.02
   Readonly: 0
   Software::License: 0.003
-  Test::YAML::Meta::Version: 0.11
+  Test::CPAN::Meta::YAML::Version: 0.11
   Text::CSV_XS: 0.45
   YAML::Syck: 0.95
   perl: 5.006
